tests/yam/agama/patch_agama_tests.pm

```diff
@@ -19,6 +19,10 @@ sub run {
 
     my ($repo, $branch) = split /#/, get_required_var('YUPDATE_GIT');
     assert_script_run("yupdate patch $repo $branch", timeout => 60);
+
+    my $agama_dir = "/usr/share/agama";
+    my $test = get_required_var('AGAMA_TEST');
+    assert_script_run("cp " . $agama_dir . "/puppeteer/build/" . $test . ".js " . $agama_dir ."/integration-tests/tests/" . $test . ".js", timeout => 30);
```

Comment:
rake should put all we need in the right directory, why we need this?

Reply:
Solved in this commit 373e1616a7b322ecb6190b84ae983c2b9da51c95